scss/techs.scss

```diff
@@ -5,39 +5,20 @@ $card-height: 350px;
 $card-width: 210px;
 
 .techs-container {
-  display: flex;
-  flex-grow: grow;
-  margin-left: 160px;
-}
-
-.main-tech-container {
-  margin-right: 30px;
-  position: relative;
-
-  &:hover {
-    .background-tech-container {
-      left: 10px;
-      top: 10px;
-    }
-  }
+  align-items: center;
+  display: grid;
+  grid-column-gap: 20px;
+  grid-template: repeat(2, $card-height) / repeat(3, $card-width);
```

Comment:
You should use `max-content` instead of the values of the card size so the container will have the maximum content avilable

Reply:
Done